src/yunohost/settings.py

```diff
@@ -30,11 +30,21 @@
 # * enum (in form a python list)
 
 # we don't store the value in default options
+PWD_MODE = ["disabled", "weak", "strong"]
+PWD_CHOICES = ["error", "warn_only", "disabled"]
+PWD_DEFAULT_ERROR = {"type": "enum", "default": "error",
+                     "choices": PWD_CHOICES}
```

Comment:
Those three variables don't seem to be used in fact ? Or am I missing something ?